website/.spelling

```diff
@@ -307,6 +307,7 @@ firefox
 firehose
 firehoses
 fromPigAvroStorage
+frontcoded
```

Comment:
is this needed?

Reply:
Huh, not sure where this came from. I just updated it to remove it and added the acronym MVDs